### PR TITLE
Hero

### PR DESCRIPTION
**Pull request by pakawaka1**

### CODE REVIEW

**Comment by pakawaka1** [comment]
@Fleury14 check out my code please.

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+.carousel-caption {
+  margin-bottom: 45px;
+}
+
+button.btn-warning {
+  background-color: #F8512F;
+  border-color: #F8512F;
+}
+
+button.btm-warning:hover {
```
Comment (Fleury14) [inline css/styles.css]: This should be btn-warning instead of btm-warning

**Review by Fleury14 — COMMENTED** [review]
Just one typo I found, otherwise looks good!

File: css/styles.css (hunk 1 above)
Comment (pakawaka1) [reply css/styles.css]: Thank you J.R.  I will fix that.

**Pull request merged**